source/plugins/languages/visual/libraries/micropython/visual/toolbox.xml

```diff
@@ -0,0 +1,513 @@
+          </block>
+        </value>
+        <value name="blue">
+          <block type="micropython_pin">
+            <field name="type1">IN</field>
+            <field name="type2">NONE</field>
+            <value name="GPIO">
+              <block type="math_number">
+                <field name="NUM">15</field>
+              </block>
+            </value>
+          </block>
+        </value>
+      </block>
+
+      <block type="micropython_rgb_pwm_set">
+        <field name="rgb">red</field>
+        <field name="duty">duty</field>
+        <value name="NAME">
+          <shadow type="math_number">
+            <field name="NUM">0</field>
+          </shadow>
+        </value>
+      </block>
+
+      <block type="micropython_rgb_pin_set">
+        <field name="rgb">red</field>
+        <field name="func">on</field>
+      </block>
+    </category>
+
+    <category name="Network" colour="#85C1E9">
+      <block type="variables_set">
+        <field name="VAR">name var</field>
+        <value name="VALUE">
+          <block type="micropython_network_wlan">
+            <field name="type">STA_IF</field>
+          </block>
+        </value>
+      </block>
+      <block type="micropython_network_set_active">
+        <field name="NAME">True</field>
+        <value name="active">
+          <block type="variables_get">
+            <field name="VAR">name var</field>
+          </block>
+        </value>
+      </block>
+      <block type="micropython_network_get_active">
+        <value name="get">
+          <block type="variables_get">
+            <field name="VAR">name var</field>
+          </block>
+        </value>
+      </block>
+      <block type="micropython_network_check_connection">
+        <field name="conn">ifconfig</field>
+        <value name="conn">
+          <block type="variables_get">
+            <field name="VAR">name var</field>
+          </block>
+        </value>
+      </block>
+      <block type="micropython_network_wifi">
+        <field name="ESSID">ESSID</field>
+        <field name="password">password</field>
+        <value name="NAME">
+          <block type="variables_get">
+            <field name="VAR">name var</field>
+          </block>
+        </value>
+      </block>
+
+      <block type="micropython_network_scan">
+        <value name="scan">
+          <block type="variables_get">
+            <field name="VAR">name var</field>
+          </block>
+        </value>
+      </block>
+      <block type="micropython_network_config">
+        <field name="NAME">mac</field>
+        <value name="config">
+          <block type="variables_get">
+            <field name="VAR">name var</field>
+          </block>
+        </value>
+      </block>
+    </category>
+
+    <category name="Print" colour="#2ECC71">
+      <block type="micropython_print"></block>
+      <block type="micropython_print_format">
+        <field name="text_quotation_mark">some text</field>
+      </block>
+    </category>
```

Comment:
This category (Print) is not necessary, these blocks are defined in the standard python block (Screen and Keyboard).